rmgpy/data/thermo.py

```diff
@@ -1660,6 +1660,68 @@ def __addPolyRingCorrectionThermoDataFromHeuristic(self, thermoData, polyring):
                 # Typically this comment is very short and also very helpful to check if the ring correction is calculated correctly.
 
 
+    def getBicyclicCorrectionThermoDataFromHeuristic(self, bicyclic):
+
+        # saturate if the bicyclic has unsaturated bonds
+        # otherwise return None
+        bicyclic_submol = convertRingToSubMolecule(bicyclic)[0]
+        saturated_bicyclic_submol, alreadySaturated = saturateRingBonds(bicyclic_submol)
+
+        if alreadySaturated:
+            return None
+        # split bicyclic into two single ring submols
+        single_ring_submols = splitBicyclicIntoSingleRings(bicyclic_submol)
+
+        # split saturated bicyclic into two single ring submols
+        saturated_single_ring_submols = splitBicyclicIntoSingleRings(saturated_bicyclic_submol)
+
+        # apply formula: 
+        # bicyclic correction ~= saturated bicyclic correction - 
+        # saturated single ring corrections + single ring corrections
+        saturated_bicyclic_thermoData = self.__addRingCorrectionThermoDataFromTree(None, 
+                self.groups['polycyclic'], saturated_bicyclic_submol, saturated_bicyclic_submol.atoms)[0]
+
+        estimated_bicyclic_thermodata = saturated_bicyclic_thermoData
+        
+        for submol in saturated_single_ring_submols:
+            
+            if not isAromaticRing(submol):
+                aromaticBonds = findAromaticBondsFromSubMolecule(submol)
```

Comment:
Why do you try to find aromatic bonds if the submol is not an aromatic ring? Also, isn't the ring saturated already?

Reply:
I saw the other commit where you don't saturate aromatic bonds anymore. However, you set the bond order of aromatic bonds to 1 here anyway, so what is the reason for not saturating them to begin with?